server/public/src/base.js

```diff
@@ -102,15 +74,14 @@ var tr = 'transform',
 function j(element) {
   return function(data) {
     var update = g.selectAll(element)
-        .data(data || g.datum());
-    update.e = update.enter()
+        .data(data || [g.datum()]);
```

Comment:
This is the only 'functional' change that I made when moving to this obfuscation method - the original code `.data(data || g.datum())` did not work.

On that note, not sure how it worked before as `data` expects an array of items.